### PR TITLE
Felipe add dataset service

### PR DESCRIPTION
**Pull request by fcoralsasso**
This PR adds datasets service to CanDIGv2 structure.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -84,4 +84,14 @@ deps=-r{toxinidir}/lib/rnaget/rnaget_service/requirements.txt
 
 commands=
     python setup.py develop
-    screen -dmS {envname} candig_rnaget --host={env:RNA_GET_HOST} --port={env:RNA_GET_PORT}
\ No newline at end of file
+    screen -dmS {envname} candig_rnaget --host={env:RNA_GET_HOST} --port={env:RNA_GET_PORT}
+
+[testenv:datasets]
+basepython=python3.7
```
Comment (shaikh-rashid) [inline tox.ini:90]: Just curious, why is python3.7 needed here if we are already using it for global?

File: tox.ini [hunk 2]
```diff
@@ -84,4 +84,14 @@ deps=-r{toxinidir}/lib/rnaget/rnaget_service/requirements.txt
...
+changedir={toxinidir}/lib/datasets/datasets_service
+deps=-r{toxinidir}/lib/datasets/datasets_service/requirements.txt
+commands=
+    ; mkdir log
```
Comment (shaikh-rashid) [inline tox.ini:94]: maybe we should start to put this tmp files in a local tmp directory? {toxinidir}/tmp/ like this?

File: lib/datasets/run.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+set -e
+
+source .env
+
+[ -z $CONDA_DEFAULT_ENV ] && echo 'Conda ENV not active!' && exit 1
+
+pushd $(pwd)/lib/datasets/datasets_service/
+    pip install -r requirements.txt
+    python -m candig_dataset_service --host ${DATASETS_SERVICE_HOST} --port ${DATASETS_SERVICE_PORT}
```
Comment (shaikh-rashid) [inline lib/datasets/run.sh:9]: shouldn't this be {env: DATASETS_SERVICE_HOST} ?

File: tox.ini (hunk 1 above)
Comment (fcoralsasso) [reply tox.ini:90]: Fixed.

File: tox.ini (hunk 2 above)
Comment (fcoralsasso) [reply tox.ini]: Fixed.

**Pull request merged**